k8s/backend.yaml

```diff
@@ -0,0 +1,601 @@
+kind: ConfigMap
+metadata:
+  name: tempo
+  namespace: observability-backend
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: tempo
+  name: tempo
+  namespace: observability-backend
+spec:
+  ports:
+  - name: tempo-prom-metrics
+    port: 3200
+    targetPort: 3200
+  - name: tempo-otlp
+    port: 55680
+    protocol: TCP
+    targetPort: 55680
+  - name: http
+    port: 80
+    targetPort: 3200
+  - name: receiver
+    port: 6831
+    protocol: UDP
+    targetPort: 6831
+  selector:
+    app: tempo
+    name: tempo
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tempo
+  namespace: observability-backend
+spec:
+  minReadySeconds: 10
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: tempo
+      name: tempo
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+  template:
+    metadata:
+      annotations:
+        config_hash: 7f4b5fad0e6364b6a2a5ea380281cb0e
+      labels:
+        app: tempo
+        name: tempo
+    spec:
+      containers:
+      - args:
+        - -config.file=/conf/tempo.yaml
+        - -mem-ballast-size-mbs=1024
+        env:
+        - name: JAEGER_AGENT_PORT
+          value: ""
+        image: grafana/tempo:main-e6394c3
+        imagePullPolicy: IfNotPresent
+        name: tempo
+        ports:
+        - containerPort: 3200
+          name: prom-metrics
+        - containerPort: 55680
+          name: otlp
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /conf
+          name: tempo-conf
+      volumes:
+      - configMap:
+          name: tempo
+        name: tempo-conf
+---
+apiVersion: v1
+data:
+  mimir.yaml: |
+    blocks_storage:
+        storage_prefix: blocks
+        tsdb:
+            dir: /data/tsdb
+    common:
+        storage:
+            filesystem:
+                dir: /data
+    ingester:
+        ring:
+            replication_factor: 1
+    multitenancy_enabled: false
+    server:
+        grpc_listen_port: 9095
+        http_listen_port: 8080
+kind: ConfigMap
+metadata:
+  name: mimir
+  namespace: observability-backend
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: mimir
+  name: mimir
+  namespace: observability-backend
+spec:
+  ports:
+  - name: mimir-prom-metrics
+    port: 8080
+    targetPort: 8080
+  - name: http
+    port: 80
+    targetPort: 8080
+  - name: grpc
+    port: 9095
+    targetPort: 9095
+  selector:
+    app: mimir
+    name: mimir
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: mimir
+  namespace: observability-backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mimir
+      name: mimir
+  serviceName: mimir
+  template:
+    metadata:
+      annotations:
+        config_hash: 379ac24872c3157214f0a463540dd3b5
+      labels:
+        app: mimir
+        name: mimir
+    spec:
+      containers:
+      - args:
+        - -config.file=/conf/mimir.yaml
+        image: grafana/mimir:2.4.0
```

Comment:
What is the reason for using the grafana's mimir project?  

I would prefer to use vanilla CNCF Prometheus server for metrics

Reply:
it was simple. The original manifest was generated using tns.